core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/cumSum.kt

```diff
@@ -18,6 +21,39 @@ import kotlin.reflect.typeOf
 
 // region DataColumn
 
+/**
+ * ## The CumSum Operation
+ *
+ * Computes the cumulative sums of the values in the each column from the {@get [CumSumDocs.DATA_TYPE]}
+ * from the first row to the last row.
```

Comment:
@zaleslaw when adding this doc to `DataColumn.cumSum` you're now talking about rows inside a data column. This is not entirely wrong, I suppose, but it looks a bit weird. Usually we're talking about rows (across multiple columns) and columns (across multiple rows). A single value in a column is either a "cell" or a "value", not a "row". right?